src/components/buttons/Buttons.jsx

```diff
@@ -19,19 +20,27 @@ const Buttons = () => {
   }));
 
   const classes = useStyles();
+  const { i18n } = useLingui();
 
   return (
-    <div>
+    <>
+    <span>Current locale: {i18n.locale}</span>
       <Button
         variant="contained"
         color="primary"
         disableElevation
         className={classes.testButtons}
       >
-        Example button Material-UI
+        <Trans id="Example button Material-UI">Example button Material-UI</Trans>
```

Comment:
text inside <Trans> tag is already an ID.  So you can do <Trans>Example button Material-UI</Trans>